shavar/templates/swagger.mako

```diff
@@ -0,0 +1,100 @@
+swagger: "2.0"
+host: ${HOST}
+info:
+  title: Shavar Service
+  description: Mozilla's implementation of the safe browsing protocol
+  contact:
+    name: Ryan Tilder
+    email: rtilder@mozilla.com
+  license:
+    name: MPL v2.0
+    url: https://www.mozilla.org/en-US/MPL/2.0/
+  version: ${VERSION}
+  x-mozilla-services:
+    homepage: https://github.com/mozilla-services/shavar
+schemes:
+    - ${SCHEME}
+paths:
+    /__heartbeat__:
+      get:
+        operationId: getHeartbeat
+        description: Performs an heartbeat on the app
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: The hearbeat was successfull
+    /__version__:
+      get:
+        operationId: getVersion
+        description: Returns the version
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: The version is returned
+    /list:
+      get:
```

Comment:
> I'm afraid if we remove the GET implementation we will find some old or obscure version/fork of Firefox that uses the GET endpoint. 😞

I don't actually think it would cause any problems because we've never used that API in Firefox.